native/KotlinIOS/KotlinIOS/ViewController.swift

```diff
@@ -32,8 +32,8 @@ class ViewController: UIViewController, ApplicationContractView {
     
     
     @IBAction func onButtonTapped() {
-        let originStation = stations[dropDown1.selectedRow(inComponent:0)]
-        let finalStation = stations[dropDown2.selectedRow(inComponent:0)]
+        let originStation = stations[departureDropDown.selectedRow(inComponent:0)]
+        let finalStation = stations[arrivalDropDown.selectedRow(inComponent:0)]
```

Comment:
Feels slightly odd to refer to these as dropdowns, are they dropdowns? picker might be better?